modules/core/src/main/java/org/apache/ignite/internal/processors/performancestatistics/package-info.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * <!-- Package description. -->
+ * Contains classes for Ignite performance statistics.
+ */
+
+package org.apache.ignite.internal.processors.performancestatistics;
```

Comment:
AFAIK we don't need package descriptions for internal packages.
Let's remove this file.

Reply:
I have added it cause wiki says to add info files despite internal it or no:
https://cwiki.apache.org/confluence/display/IGNITE/How+to+Contribute#HowtoContribute-Creation
https://cwiki.apache.org/confluence/display/IGNITE/How+to+Contribute#HowtoContribute-package-info.java

Reply:
We don't have this file for any other internal package.
So I think it's OK to remove it here.

Reply:
Ok. Removed.